Utilities/Certificates/create-trusted-certificate.bat

```diff
@@ -0,0 +1,48 @@
+@echo off
```

Comment:
There are not many bat files in the repo but the others use CamelCasing so this could too.